subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/ResolvedGeneratedJarsIntegrationTest.groovy

```diff
@@ -40,7 +40,6 @@ class ResolvedGeneratedJarsIntegrationTest extends BaseGradleImplDepsIntegration
 
         then:
         file("user-home/caches/${distribution.version.version}/generated-gradle-jars/gradle-api-${distribution.version.version}.jar").assertExists()
-        assertSingleGenerationOutput(output, API_JAR_GENERATION_OUTPUT_REGEX)
```

Comment:
I changed this and some other tests to not check log output, because that's not relevant for what these tests are trying to verify.